Example/Emission.xcodeproj/project.pbxproj

```diff
@@ -222,12 +225,15 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 510DCB3F1CCA69ED0075E8CB /* Build configuration list for PBXNativeTarget "EmissionTests" */;
 			buildPhases = (
+				20B6FFACE9798C3F6AEDEF21 /* [CP] Check Pods Manifest.lock */,
```

Comment:
Yeah I created this branch of the latest of master, cherry picked commits (not the pod change commits), and then still had to run a pod install which resulted in these changes. That ok?